day_two_guide/topics/entropy.adoc

```diff
@@ -0,0 +1,56 @@
+////
+Entropy
+
+Module included in the following assemblies:
+
+* day_two_guide/run_once_tasks.adoc
+////
+
+{product-title} uses entropy to generate random numbers for objects such as IDs
+or SSL traffic. These operations wait until there is enough entropy to complete
+the task. Without enough entropy, the kernel is not able to generate these
+random numbers with sufficient speed, which can lead to timeouts and the refusal
+of secure connections.
+
+To check available entropy:
+
+----
+$ cat /proc/sys/kernel/random/entropy_avail
+2683
+----
+
+The available entropy should be verified on all node hosts in the cluster.
+Ideally, this value should be above `1000`.
+
+[NOTE]
+====
+Red Hat recommends monitoring this value and issuing an alert if the value is
+under `800`.
+====
+
+The most reliable way to determine the available entropy is with the
+`rngtest` command from the `rng-tools`: 
+
+----
+$ cat /dev/random | rngtest -c 100
+----
+
+If the above takes too long to complete, then there is not enough entropy
+available.
+
+Depending on your environment, entropy can be increased in multiple ways. For
+more information, see the following blog post:
+https://developers.redhat.com/blog/2017/10/05/entropy-rhel-based-cloud-instances/
+
+Generally, you can increase entropy by installing the `rng-tools` package and
+enabling the `rngd` service:
+
+----
+# yum install rng-tools
```

Comment:
yum install rng-tools -y

Reply:
@e-minguez We shy against using the "-y" option in docs to give the reader that extra bit to back away.

Reply:
ok!